### ISSUE TITLE
[BUG] Set the lore of a loop-item ?

### ISSUE THREAD

**Issue by Zerkhan** [Njol/Skript#310]
Hi Njol,

I have downloaded Skript 2.1.1 (https://dl.dropboxusercontent.com/u/48570304/Skript/Skript-2.1.1.jar), and I have write this :

```
command /test:
    trigger:
        give iron sword of sharpness 1 named "NAME" to player
        loop all items in player's inventory:
            loop-item's name is "NAME":
                # The text is broadcasted.
                broadcast "1"
                # ... but the lore isn't set D:
                set line 1 of loop-item's lore to "LORE"
```

I have also tried with,

```
set {_p} to "LORE"
set line 1 of loop-item's lore to {_p}
```

```
set {_p} to "LORE" parsed as text
set line 1 of loop-item's lore to {_p}
```

May be you are already aware of this problem, sorry in this case.

**Comment by Njol** [comment]
No, I was not aware of this. This is now officially the last bug fixed for 2.1.1, as I just uploaded it to Bukkit.

**Comment by Zerkhan** [comment]
Yeah \o/ !
